src/test/java/Homework18.java

```diff
@@ -7,9 +7,7 @@ public class Homework18 extends BaseTest {
 
     @Test
     public void playSong() throws InterruptedException {
-        provideEmail("demo@class.com");
-        providePassword("te$t$tudent");
-        clickSubmitBtn();
+        login();
```

Comment:
Try to refactor your code because the music is playing now, when the soundbar is visible

![image](https://user-images.githubusercontent.com/104761482/200293765-558b62f2-b3f2-4a79-9cb5-af2278a79be4.png)


Here is an example

![image](https://user-images.githubusercontent.com/104761482/200293207-68c00ede-0240-4ae9-9f46-0b23ef023909.png)